src/applications/static-pages/simple-forms/40-0247/App.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+
+const App = ({ formEnabled }) => {
+  if (formEnabled === undefined) {
+    return <va-loading-indicator message="Loading..." />;
+  }
+
+  if (formEnabled) {
+    return (
+      <>
+        <p>You can apply for a PMC in any of these 5 ways.</p>
+        <h3>Option 1: Apply online</h3>
+        <a
+          className="vads-c-action-link--blue"
+          href="/burials-memorials/memorial-items/presidential-memorial-certificates/request-certificate-form-40-0247"
+        >
+          Apply for a PMC online
+        </a>
+        <h3>Option 2: Mail your application</h3>
+        <p>
+          First, complete the Presidential Memorial Certificate Request Form (VA
+          Form 40-0247).
+        </p>
+        <p>
+          <va-link
+            href="/find-forms/about-form-40-0247"
+            text="Get VA Form 40-0247 to download"
+          />
+        </p>
+        <p>
+          Mail your application, along with copies of the Veteran’s death
+          certificate and DD214 or other discharge documents, to this address:
+        </p>
+        <p>
+          NCA FP Evidence Intake Center
```

Comment:
is it possible to use the address component here? @tlei123 @tbaker1026 

https://design.va.gov/components/address-block

Reply:
@megzehn , yes; className="va-address-block" added to the <p> tag.  It'll looks like an [address block](https://design.va.gov/storybook/?path=/docs/components-address-block--default-story) on your page.